Profiles/BCU/responses.yml

```diff
@@ -88,6 +88,9 @@ Chat:
   TrackerGuessFailed:
   - Text: I might have lost, but you're still not getting my soul.
   - Text: I guess I lost. I don't have a soul for you to steal though.
+  PollErrorTerminated:
+  - Text: Did you think I wouldn't catch on to what you've been doing with the content polls, s kennedy? I hope you know I'm recording this to my personal stat tracking sheet.
+    Weight: 50 # Remove after it comes up
```

Comment:
We really should build something for this case, honestly.

Reply:
Yeah, I could see it being a useful thing. I especially get a bit concerned because some of these (like this one) are *very* topical to betus only. So would be good if with that we could come up with a way to isolate responses to just betus as well.